src/common/htmlFileReader.test.ts

```diff
@@ -128,4 +129,56 @@ describe("Html File Reader", () => {
             expect(axios.get).toBeCalledWith(asset.path, { responseType: "blob" });
         });
     });
+
+    describe("Test non valid asset type", () => {
```

Comment:
do you need both describe and it, they have same text

Reply:
I think in general it's a good practice to always group tests around a similar scenario even when we have a single test in that category.  It's a good practice as you may want to add more tests on that category/describe later but in this specific case I don't see other possible tests there so I removed the describe call.